packages/tpetra/core/ext/TpetraExt_TripleMatrixMultiply_def.hpp

```diff
@@ -42,6 +42,7 @@
 #define TPETRA_TRIPLEMATRIXMULTIPLY_DEF_HPP
 
 #include "TpetraExt_MatrixMatrix_def.hpp"
+#include "TpetraExt_MatrixMatrix_ExtraKernels_decl.hpp" //for UnmanagedView
```

Comment:
Maybe think about separating UnmanagedView into a separate header file?  It sounds like a useful thing that ultimately Kokkos needs.